modules/ai-install-ocp-clusters-on-bare-metal.adoc

```diff
@@ -0,0 +1,216 @@
+// TELCODOCS-272 4.8 Using Assisted Installer to Install & Manage ocp Clusters on Bare Metal
+//
+//
+//
+= Enabling Assisted Installer Service on Bare Metal
+<<introduction,Introduction to Assisted Installer Service>>
+<<enable-assisted-installer-service,Enabling the Assisted Installer Service>>
+<<resources-explained,Resources Explained>>
+
+== Introduction
+
+The Assisted Installer Service (AIS) deploys Red Hat OpenShift Container Platform (ocp) clusters running on a Bare Metal Operator (BMO) node. Red Hat Advanced Cluster Management (RHACM) 2.3+ ships with AIS, which deploys once the `MultiClusterHub` is enabled.
+
+RHACM 2.3+ only supports *Single Node OpenShift (SNO)* cluster deployments. A SNO cluster runs OpenShift Container Platform (ocp) on top of a single BMO node. The single BMO acts as *both the supervisor and the worker node*.
+
+== Enable assisted installer service
+
+=== Prerequisites
```

Comment:
s\=== Prerequisites\.Prerequisites 

In general, you have too many section titles in the draft. https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#writing-procedures

"When needed, use .Prerequisites, .Next steps, or .Additional resources syntax to suppress TOC formatting within a module. Do not use == headings or xrefs in modules."

Reply:
Updated syntax as suggested.